llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -370,7 +370,7 @@ class MemoryDepChecker {
     /// Strides could either be scaled (in bytes, taking the size of the
     /// underlying type into account), or unscaled (in indexing units; unscaled
     /// stride = scaled stride / size of underlying type). Here, strides are
-    /// unscaled.
+    /// scaled.
```

Comment:
You could probably remove the description of "unscaled", since all the strides are now in byte units.